main.py

```
@@ -40,26 +42,80 @@ def main(**args):
+                dxpy.bindings.find_data_objects(
+                    project=sd_wgs_project.id,
+                    created_after=f"-{args['time_to_check']}",
+                )
             )
 
+        if not all(
+            check.check_file_input_name_is_correct(file, patterns)
+            for file in new_files
+        ):
+            raise AssertionError(
+                f"The set of provided files is not correct. Expected files with the following patterns: {[
```

Pylint (3.3.7)

[error] Parsing failed: 'unterminated string literal (detected at line 107) (main, line 107)'

(E0001)